Sources/MapboxNavigation/NavigationOptions.swift

```diff
@@ -61,15 +103,17 @@ open class NavigationOptions: NavigationCustomizable {
      - parameter topBanner: The container view controller that presents the top banner.
      - parameter bottomBanner: The container view controller that presents the bottom banner.
      - parameter predictiveCacheOptions: Configuration for predictive caching. These options control how the `PredictiveCacheManager` will try to proactively fetch data related to the route. A `nil` value disables the feature.
+     - parameter tileStoreLocation: TODO
      */
-    public convenience init(styles: [Style]? = nil, navigationService: NavigationService? = nil, voiceController: RouteVoiceController? = nil, topBanner: ContainerViewController? = nil, bottomBanner: ContainerViewController? = nil, predictiveCacheOptions: PredictiveCacheOptions? = nil) {
+    public convenience init(styles: [Style]? = nil, navigationService: NavigationService? = nil, voiceController: RouteVoiceController? = nil, topBanner: ContainerViewController? = nil, bottomBanner: ContainerViewController? = nil, predictiveCacheOptions: PredictiveCacheOptions? = nil, tileStoreLocation: TileStoreLocation = .default) {
```

Comment:
There are a lot of arguments with default values. If the default is sensible in most cases, we could omit this argument from the initializer and expect developers to set the `tileStoreLocation` property on a separate line.

This goes for the rest of the arguments too. I’m not sure why we really need a convenience initializer, since there’s no inherent order to the arguments other than the order in which we implemented each feature. v2.0 would be a good opportunity to remove this initializer, though maybe in a separate PR.